src/EnergyPlus/WeatherManager.cc

```diff
@@ -96,35 +96,38 @@ namespace WeatherManager {
     //       AUTHOR         Rick Strand
     //       DATE WRITTEN   May 1997
     //       MODIFIED       December 1998, FW; December 1999, LKL.
-    //       RE-ENGINEERED  na
 
     // PURPOSE OF THIS MODULE:
     // This module contains all of the weather handling routines for
     // EnergyPlus.  That includes getting user input, defining design day
     // weather, retrieving data from weather files, and supplying the
     // outdoor environment for each time step.
 
-    // METHODOLOGY EMPLOYED:
-    // Setting up the design days is similar to BLAST/IBLAST.  Reading the
-    // BLAST weather files is similar to that code in BLAST/IBLAST.  The EnergyPlus
-    // Weather file (EPW) is new code.
+    constexpr std::array<std::string_view, static_cast<int>(EpwHeaderType::Num)> epwHeaders({"LOCATION",
```

Comment:
Moved this here and made it a std::array, which led to some squirrel chases.

Reply:
Fun!